server/events/approve_policies_command_runner.go

```diff
@@ -91,9 +95,23 @@ func (a *ApprovePoliciesCommandRunner) buildApprovePolicyCommandResults(ctx *com
 	// Check if vcs user is in the owner list of the PolicySets. All projects
 	// share the same Owners list at this time so no reason to iterate over each
 	// project.
-	if len(prjCmds) > 0 && !prjCmds[0].PolicySets.IsOwner(ctx.User.Username) {
-		result.Error = fmt.Errorf("contact policy owners to approve failing policies")
-		return
+	if len(prjCmds) > 0 {
+		teams := []string{}
+
+		// Only query the users team membership if any teams have been configured as owners.
+		if prjCmds[0].PolicySets.HasTeamOwners() {
```

Comment:
nice check 👍 

📝 if `teams` setting is empty, not call api. so no breaking changes.